src/components/popover-menu/popover-menu-panel-group.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+interface PopoverMenuGroupProps {
+    children: React.ReactNode;
```

Comment:
Is the pattern of using the types as `React.[Type]` a hailstorm pattern? In the app codebase we usually import the type deestructured from React. If it's a pattern in the codebase leave it as is, if it's different we should be consistent to whatever the pattern is in this codebase.

Reply:
yes, in hailstorm this is the standard :-)